presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSource.java

```diff
@@ -134,15 +95,16 @@ public Page getNextPage()
                 return null;
             }
 
-            Block[] blocks = new Block[hiveColumnIndexes.length];
-            for (int fieldId = 0; fieldId < blocks.length; fieldId++) {
-                if (constantBlocks[fieldId] != null) {
-                    blocks[fieldId] = constantBlocks[fieldId].getRegion(0, batchSize);
```

Comment:
We're no longer reusing `constantBlocks`. I think this is OK. @dain

Reply:
For both this and the streaming, I'm not super concerned as this code is only executed once per page.  That said, we haven't measured the impact, and the new code isn't dramatically more readable, so I'd would have left is as is to keep risk down.  I don't have a strong preference, so I open to either solution.